eZ/Bundle/EzPublishCoreBundle/Imagine/Cache/Resolver/ProxyResolver.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine\Cache\Resolver;
+
+use Liip\ImagineBundle\Imagine\Cache\Resolver\ProxyResolver as ImagineProxyResolver;
+
+class ProxyResolver extends ImagineProxyResolver
+{
+    /**
+     * Replaces host with given proxy host.
+     *
```

Comment:
maybe add some doc here on why this method is overloaded. maybe even short diff compared to liip for future maintenance know-how.

Reply:
Added short info in https://github.com/ezsystems/ezpublish-kernel/pull/2604/commits/5af6ddf46b4befbb8e262d9b90f07b4966fdcd73.